lib/matplotlib/tests/test_axes.py

```diff
@@ -6372,3 +6372,11 @@ def test_polar_interpolation_steps_variable_r(fig_test, fig_ref):
     l.get_path()._interpolation_steps = 100
     fig_ref.add_subplot(projection="polar").plot(
         np.linspace(0, np.pi/2, 101), np.linspace(1, 2, 101))
+
+
+@image_comparison(baseline_images=['tickcolor_is_not_markercolor'],
+                  remove_text=True,
+                  extensions=['png'])
+def test_tickcolor_is_not_markercolor():
```

Comment:
Not sure you need an image test for this; you should be able to check tick properties are what you want instead.

Reply:
Thank you @QuLogic, I fixed this now and reverted the commit that added the baseline image. Apologies for the late reply. I only saw your comment now.